docs/learn/install.md

```diff
@@ -286,11 +260,25 @@ m64/ 12()core 65536MB sjt mackenzie.local 127.0.0.1 EXPIRE…
 q)
 ```
 
-:fontawesome-solid-laptop-code:
-[Installing multiple versions of kdb+](../kb/versions.md)
+## Further customization
+
+### rlwrap for Linux and macOS
 
+On Linux and macOS the `rlwrap` command allows the Up arrow to retrieve earlier expressions in the q session. It is very useful and we recommend installing it.
```

Comment:
```suggestion
On Linux and macOS, the `rlwrap` command allows the Up arrow to retrieve earlier expressions in the q session. This can be very useful and it is recommended you install it.
```